vocabularies/Common.json

```diff
@@ -1282,6 +1282,14 @@
       "@Common.Experimental": true,
       "@Core.IsURL": true,
       "@Core.Description": "Base URL for WebSocket connections"
+    },
+    "ValidationFunction": {
+      "$Kind": "Term",
+      "$Type": "Common.QualifiedName",
+      "$AppliesTo": ["Parameter", "Action", "Function"],
```

Comment:
Does it make sense to introduce a ValidationFunction for a Function? After all, the function can simply be executed and it would return whether parameter values are good (together with the respective error messages).

Reply:
This annotation can be used to validate individual function parameters even before all parameters have been entered.

Reply:
Yes that's the whole point :)